src/widgets/ot_pulltorefresh.eliom

```diff
@@ -0,0 +1,261 @@
+[%%shared.start]
+
+[%%client 
+  open Eliom_content.Html
+]
+
+open Eliom_content.Html.D
+
+[%%client
+
+  module type PULLTOREFRESH = sig
+    val dragThreshold : float 
+    val moveCount : int
+    val headContainerHeight : int
+    val pullDownIcon : Html_types.div  Eliom_content.Html.D.elt
+    val loadingIcon : Html_types.div  Eliom_content.Html.D.elt
+    val successIcon : Html_types.div  Eliom_content.Html.D.elt
+    val failureIcon : Html_types.div  Eliom_content.Html.D.elt
+    val pullText :  Html_types.span  Eliom_content.Html.D.elt 
+    val container :  Html_types.div  Eliom_content.Html.D.elt
+    val pullDownText : string
+    val releaseText : string
+    val loadingText : string
+    val successText : string
+    val failureText : string
+    val rotateGradually : bool
+    val blockPullIcon : bool
+    val afterPull : unit -> bool Lwt.t
+  end
```

Comment:
I would `PULLTOREFRESH` rather call `CONF` (or `CONFIG` or `CONFIGURATION`). Since we are in a structured namespace `Ot_pull_to_refresh.CONF` is more descriptive than `Ot_pull_to_refresh.PULLTOREFRESH`.